test/libsolidity/syntaxTests/inlineAssembly/two_stack_slots_calldata.sol

```diff
@@ -0,0 +1,10 @@
+contract C {
+    function f() pure external {
+        function() external two_stack_slots;
```

Comment:
How is this test calldata specific?

Reply:
I think I added this test to trigger error nr 9857 - will rename the file.

Reply:
Did you rename the file?